vscode/package.json

```diff
@@ -511,17 +538,18 @@
     ],
     "debuggers": [
       {
-        "type": "qsharp",
+        "type": "qdk",
```

Comment:
Since the debug type may have been persisted to a launch.json, I'm not sure if it's a good idea to change its name now.